scripts/zones/Yuhtunga_Jungle/mobs/Carthi.lua

```diff
@@ -15,18 +15,18 @@ end;
 -----------------------------------
 
 function onMobDeath(mob, player, isKiller)
+    if (isKiller) then -- Causes this to only run once per Carthi kill
```

Comment:
My concern with is approach is, if it dies to DoT while unclaimed then this won't run. Does it have to die claimed on retail to work or no?